02 - Memtables, Joins, and Selectors.ipynb

```diff
@@ -463,7 +463,7 @@
    "outputs": [],
```

Comment:
<p>add <code>from ibis import _</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis-tutorial/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='98'/>

Reply:
<p>It's earlier in the notebook, but yeah, I'll add it in again because of needing to restart the notebook server (and I'll figure out how to avoid that...)</p>